source/sass/buyers-guide/fragments/_draft_product.scss

```diff
@@ -0,0 +1,14 @@
+// product draft rules
+&.draft::before {
+  content: 'This is a draft.';
```

Comment:
Although not a huge problem in this case - this is bad for accessibility as content generated in CSS pseudo selectors don't usually get picked up by screen reader. I'd suggest that we use JS to inject a node for this "This is a draft." note instead.

Reply:
Given that this is _not_ for real users, I'm not too concerned with this. I'd rather keep it "out of content" until it becomes necessary to make it real content, since it's purely a visual cue, not "page content".

Reply:
/cc @kristinashu @alanmoo

Reply:
I get both sides, but yeah, we already use pseudoelements to indicate draft pages on opportunity pages so let's go with this as-is.